src/main/java/org/jetbrains/bsp/bazel/server/bsp/services/JavaBuildServerService.java

```diff
@@ -1,80 +1,43 @@
 package org.jetbrains.bsp.bazel.server.bsp.services;
 
-import ch.epfl.scala.bsp4j.BuildTargetIdentifier;
 import ch.epfl.scala.bsp4j.JavacOptionsItem;
 import ch.epfl.scala.bsp4j.JavacOptionsParams;
 import ch.epfl.scala.bsp4j.JavacOptionsResult;
 import com.google.common.collect.ImmutableList;
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 import org.eclipse.lsp4j.jsonrpc.messages.Either;
 import org.eclipse.lsp4j.jsonrpc.messages.ResponseError;
 import org.jetbrains.bsp.bazel.commons.Constants;
-import org.jetbrains.bsp.bazel.commons.Uri;
+import org.jetbrains.bsp.bazel.server.bazel.BazelRunner;
 import org.jetbrains.bsp.bazel.server.bazel.data.BazelData;
-import org.jetbrains.bsp.bazel.server.bsp.resolvers.ActionGraphParser;
-import org.jetbrains.bsp.bazel.server.bsp.resolvers.ActionGraphResolver;
 import org.jetbrains.bsp.bazel.server.bsp.resolvers.TargetsResolver;
 
 public class JavaBuildServerService {
 
-  private final BazelData bazelData;
-  private final TargetsResolver targetsResolver;
-  private final ActionGraphResolver actionGraphResolver;
-
-  public JavaBuildServerService(
-      BazelData bazelData,
-      TargetsResolver targetsResolver,
-      ActionGraphResolver actionGraphResolver) {
-    this.bazelData = bazelData;
-    this.targetsResolver = targetsResolver;
-    this.actionGraphResolver = actionGraphResolver;
+  private static final String JAVA_COMPILER_OPTIONS_NAME = "javacopts";
```

Comment:
we're moving these constants back and forth :P maybe let's keep them in Constants.java

Reply:
I feet that this is the proper final place for this constant